### PR TITLE
Updates to use v3.1 of the Regalii API

### PR DESCRIPTION
**Pull request by leighhalliday**

### CODE REVIEW

**Comment by leighhalliday** [comment]
Here's an example script I ran to make sure the changes work... no tests unfortunately for now:
```php
<?php
require __DIR__ . '/vendor/autoload.php';

$configuration = new Regaliator\Configuration([
  'version' => '3.1',
  'api_host' => 'https://test.casiregalii.com',
  'api_key' => getenv('REGALII_API_KEY'),
  'secret_key' => getenv('REGALII_SECRET')
]);
$regaliator = new Regaliator\Regaliator($configuration);

// Success
$response = $regaliator->create_credentials_bill(12376, 'login', 'password');
var_dump(json_decode($response->body, true));

// MFA Question Challenge
$response = $regaliator->create_credentials_bill(12376, 'login', 'challengeme');
$bill = json_decode($response->body, true);

$response = $regaliator->update_bill_mfas($bill['id'], ['mfa_challenges' => [
  [
    'id' => $bill['mfa_challenges'][0]['id'],
    'type' => $bill['mfa_challenges'][0]['type'],
    'response' => '8'
  ]
]]);
var_dump(json_decode($response->body, true));

// Slow
$response = $regaliator->create_credentials_bill(12376, 'slow', 'slow');
var_dump(json_decode($response->body, true));

// Invalid Credentials
$response = $regaliator->create_credentials_bill(12376, 'credentials', 'credentials');
var_dump(json_decode($response->body, true));

// Show Bill
$response = $regaliator->show_bill(775);
var_dump(json_decode($response->body, true));

// Refresh
$response = $regaliator->refresh_bill(777);
var_dump(json_decode($response->body, true));
```

File: README.md [hunk 1]
```diff
@@ -7,16 +7,20 @@ A PHP client for connecting to the Regalii API.
...
 
 ```php
-$regaliator = new Regaliator\Regaliator(
-  'https://test.casiregalii.com',
-  getenv('REGALII_API_KEY'),
-  getenv('REGALII_SECRET')
-);
+$configuration = new Regaliator\Configuration([
+  'version' => '3.1',
+  'api_host' => 'https://test.casiregalii.com',
```
Comment (GRoguelon) [inline README.md]: FYI on the Ruby client, we doesn't specify the protocol only the domain name, i.e: `test.casiregalii.com`.

**Review by GRoguelon — APPROVED** [review]
That seems perfect.

**Comment by GRoguelon** [comment]
👍

**Pull request merged**